pyiron_base/generic/hdfio.py

```diff
@@ -111,6 +111,8 @@ class FileHDFio(object):
     """
 
     def __init__(self, file_name, h5_path="/", mode="a"):
+        if not file_name.endswith(".h5"):
```

Comment:
How do you feel about this? I'm not actually 100% sure it's any clearer, but it has the (small) advantage of being 50% shorter in line-space 😉 

```python
file_name += ".h5" if not file_name.endswith(".h5") else ""
```

Reply:
Purely aesthetically I like the current version more, but I don't mind changing it.